### ISSUE TITLE
[Feature request] Add Undo option on Congratulation screen

### ISSUE THREAD

**Issue by snowtimeglass** [ankidroid/Anki-Android#15482]
**Is your feature request related to a problem? Please describe.**

Users can't do Undo action on Congratulation screen on 2.17beta2.

###### Expected Result
Users can do Undo action on Congratulation screen, as in Anki Desktop.

**Describe the solution you'd like**
Add Undo option on App bar of Congratulation screen.

**Describe alternatives you've considered**
N/A

**Additional context**
Current indirect way: go back to Deck picker screen > three dot menu > 'Undo ~'

**Comment by BrayanDSO** [comment]
>  as in Anki Desktop.

How exactly? Besides by using keyboard shortcuts. I don't find any button there

![image](https://github.com/ankidroid/Anki-Android/assets/69634269/bd739832-3be3-41bc-984a-ab7070420bfc)

**Comment by snowtimeglass** [comment]
>>as in Anki Desktop.
>
>How exactly? Besides by using keyboard shortcuts. I don't find any button there

No, I don't find it, either. I mean "In Anki Desktop, users can do Undo action on Congratulation screen by using keyboard shortcuts".

**Comment by devharsh2k4** [comment]
can i get assigned??

**Comment by BrayanDSO** [comment]
This screen needs a revamp and it isn't certain if there is going to be an undo button or not, so I'll close this and leave the discussion to another more general issue about the screen (which should take time since it the priority is low)
